modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -105,6 +105,20 @@ necessary IAM policies required to run the agent.
 
 NOTE: With standard BYOC clusters, Redpanda manages security policies and resources for your VPC, including subnetworks, service accounts, IAM roles, firewall rules, and storage buckets. For the most security, you can manage these resources yourself with a xref:./vpc-byo-gcp.adoc[customer-managed VPC].
 
+== Cluster tiers
+
+When you create a cluster, you select a throughput tier for your expected ingress and egress. You can scale up and down from one tier to another with minimal downtime. The maximum number of partitions and connections also increases with each tier. For example:
```

Comment:
Instead of "For example:" I think we should be more definitive, something along the lines of: "The current defined limits for each tier are as follows"